cluster/gce/gci/configure.sh

```diff
@@ -546,6 +547,42 @@ function install-containerd-ubuntu {
   sudo systemctl start containerd
 }
 
+function install-auth-provider-gcp {
+  local -r auth_provider_tar="auth-provider-gcp-${HOST_PLATFORM}_amd64.tar.gz"
```

Comment:
Can we include a version (or git hash of some sort the binary was built from) in the filename in the gcs bucket? And maybe also add the DEFAULT_AUTH_PROVIDER_VERSION similar as we have `DEFAULT_NPD_VERSION`. That will allow to easily bump the version if we need.